vizro-core/examples/_dev/app.py

```diff
@@ -0,0 +1,23 @@
+"""Example to show simple dashboard configuration."""
+
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+
+df = px.data.iris()
+
+page = vm.Page(
+    title="Page Title",
+    components=[
+        vm.Graph(id="scatter_chart", figure=px.scatter(df, x="sepal_length", y="petal_width", color="species")),
+        vm.Graph(id="hist_chart", figure=px.histogram(df, x="sepal_width", color="species")),
```

Comment:
```suggestion
        vm.Graph(figure=px.scatter(df, x="sepal_length", y="petal_width", color="species")),
        vm.Graph(figure=px.histogram(df, x="sepal_width", color="species")),
```
A small simplification I think we should make in many places (our examples and in our docs) - no need to specify the `id` here.